app/controllers/coronavirus_pages_controller.rb

```diff
@@ -46,7 +46,7 @@ def coronavirus_page
   end
 
   def updater
-    CoronavirusPages::Updater.new(slug)
+    CoronavirusPages::ModelBuilder.new(slug)
```

Comment:
I don't think it's used in multiple places yet, but is it worth memoizing to prevent potentially re-instantiating multiple times in future work?